magento-research/pwa-studio
# [ux]: Slide filter from side and restrict width

## Description

The filter modal took the entire width of the screen in desktop and it slid up from the bottom.
@soumya-ashok wants it to have the same treatment as the nav menu.

## Related Issue





Closes #1477 .

## Verification Steps
1. Open a category page.
2. Click Filter, see it slide in from left.
3. Close Filter Modal, see it slide back to left.

## Screenshots / Screen Captures (if appropriate)
[![Image from Gyazo](https://i.gyazo.com/0d1b8c84f2faec956ebb750026d109b1.gif)](https://gyazo.com/0d1b8c84f2faec956ebb750026d109b1)

## Proposed Labels for Change Type/Package




## Checklist:
